Improvements to VPN documentation.

meitar opened the pull request:
This pull request coalesces my work on the VPN portion of the NYC Mesh documentation. My goals for this chunk of work are:

* Improve beginner-level comprehension regarding the purpose of the NYC Mesh VPN service.
* Clarify existing instructions for end-user connection and setup procedures.
* Expand existing documentation to cover additional device/OS combinations.
* Add visuals such as example screenshots.
* Offer suggestions for improved formatting (using Hugo features such as relative links) and general readability (e.g., more "scannable" text with Markdown-formatted lists, headers, etc).

Please feel free to merge this PR at whatever point is comfortable as it will not include any code changes beyond content-level links and formatting. I.e., no CSS/templating changes will be commited to the branch on which this PR is based.

olivernyc commented on netlify.toml:
```diff
@@ -0,0 +1,2 @@
+[context.deploy-preview.environment]
```
If I'm not mistaken, this will only work in the deploy preview. To apply to all contexts, including production, this should be something like:
```toml
[build.environment]
  HUGO_VERSION = "0.57.2"
```

meitar replied:
That is correct but, as promised, I did not want to change your production environment in this branch (at least not without an explicit green light to do so). If you're comfortable with changing the Netflify production environment, I think it would be a smart move since Hugo's latest features are quite useful. :)

meitar commented:
@olivernyc I've addressed your feedback regarding the Netlify build in commit ba7591a84ab322eb3dd92e890b4aa2b561897f28, please have a second look?

Also, I'm going to more or less freeze my work on the VPN documentation for now and shift my focus to other parts of the docs. To that end, I've removed the "WIP" (Work In Progress) marker from the title of this pull request. I consider this "ready to merge" with your approval.

olivernyc commented:
Looks good to me, thanks!
